packages/unenv-preset/src/preset.ts

```diff
@@ -64,29 +65,25 @@ export const cloudflare: Preset = {
 		),
 
 		// The `node:sys` module is just a deprecated alias for `node:util` which we implemented using a hybrid polyfill
-		sys: "@cloudflare/unenv-preset/runtime/node/util/index",
-		"node:sys": "@cloudflare/unenv-preset/runtime/node/util/index",
+		sys: "@cloudflare/unenv-preset/node/util",
+		"node:sys": "@cloudflare/unenv-preset/node/util",
 
 		// define aliases for hybrid modules
 		...Object.fromEntries(
 			hybridNodeCompatModules.flatMap((m) => [
-				[m, `@cloudflare/unenv-preset/runtime/node/${m}/index`],
-				[`node:${m}`, `@cloudflare/unenv-preset/runtime/node/${m}/index`],
+				[m, `@cloudflare/unenv-preset/node/${m}`],
+				[`node:${m}`, `@cloudflare/unenv-preset/node/${m}`],
 			])
 		),
-
-		// TODO: this is a hotfix and breaks unenv/fetch
-		// https://github.com/unjs/unenv/issues/364
-		"unenv/runtime/node/stream/index": "node:stream",
```

Comment:
How was this resolved? I see that the polyfill was removed in https://github.com/unjs/unenv/pull/411 but it is difficult to see how that fixes this.

Reply:
unenv used to import from `unenv/runtime/node/stream/index` to get a stream implementation needed for direct fetch. It was updated to import from `node:stream`.
This is why we do need this alias to use the workerd implementation of the stream module.